tests/test-vector-ops.js

```diff
@@ -1,93 +1,83 @@
-require(["jquery", "VectorOps"], function($, VectorOps) {
-    $(document).ready(function() {
+require(["jquery", "VectorOps"], function ($, VectorOps) {
+    $(document).ready(function () {
         // Setup test variables
-        module("VectorOps" , {
-            setup: function() {
-            },
+        module("VectorOps", {
+            setup: function () {},
 
-            teardown: function() {
-            }
+            teardown: function () {},
         });
 
         // tests the constructor of bp tree
-        test("Test getAngle", function() {
+        test("Test getAngle", function () {
             // point on the positive x-axis
-            deepEqual(VectorOps.getAngle([1,0]), {"cos" : 1, "sin" : 0});
-            deepEqual(VectorOps.getAngle([100,0]), {"cos" : 1, "sin" : 0});
+            deepEqual(VectorOps.getAngle([1, 0]), { cos: 1, sin: 0 });
+            deepEqual(VectorOps.getAngle([100, 0]), { cos: 1, sin: 0 });
 
             //point on the negative x-axis
-            deepEqual(VectorOps.getAngle([-1,0]), {"cos" : -1, "sin" : 0});
+            deepEqual(VectorOps.getAngle([-1, 0]), { cos: -1, sin: 0 });
 
             // point on the positive y-axis
-            deepEqual(VectorOps.getAngle([0,1]), {"cos" : 0, "sin" : 1});
+            deepEqual(VectorOps.getAngle([0, 1]), { cos: 0, sin: 1 });
 
             // point on the negative y-axis
-            deepEqual(VectorOps.getAngle([0,-1]), {"cos" : 0, "sin" : -1});
+            deepEqual(VectorOps.getAngle([0, -1]), { cos: 0, sin: -1 });
 
             // arbitrary point
             var angle = VectorOps.getAngle([1, 1]);
             ok(Math.abs(angle.cos - Math.sqrt(2) / 2 < 1.0e-15));
             ok(Math.abs(angle.sin - Math.sqrt(2) / 2 < 1.0e-15));
 
             // arbitrary point
-            angle = VectorOps.getAngle([-5*Math.sqrt(3) / 2, -5 / 2]);
-            ok(Math.abs(angle.cos - (-1*Math.sqrt(3))/2) < 1.0e-15);
-            ok(Math.abs(angle.sin - (-1/2)) < 1.0e-15);
+            angle = VectorOps.getAngle([(-5 * Math.sqrt(3)) / 2, -5 / 2]);
+            ok(Math.abs(angle.cos - (-1 * Math.sqrt(3)) / 2) < 1.0e-15);
+            ok(Math.abs(angle.sin + 1 / 2) < 1.0e-15);
```

Comment:
```suggestion
            ok(Math.abs(angle.sin - (-(1 / 2)) < 1.0e-15);
```
Again not a huge issue so if the linter complains don't worry about it (I think the linter will complain seeing this change was the original line)